test/emqx_access_rule_SUITE.erl

```diff
@@ -44,10 +44,18 @@ t_compile(_) ->
     Rule4 = {allow, {'or', [{client, all}, {user, all}]}, pubsub, [ <<"testTopics1">>,  <<"testTopics2">>]},
     Compile4 = {allow, {'or', [{client, all}, {user, all}]}, pubsub, [[<<"testTopics1">>],  [<<"testTopics2">>]]},
 
+    Rule5 = {allow, {ipaddrs, ["127.0.0.1", "192.168.1.0/24"]}, pubsub, <<"%c">>},
+    Compile5 = {allow, {ipaddrs,[{{127,0,0,1},{127,0,0,1},32},
```

Comment:
maybe `ip_cidr` instead `ipaddrs` is just one letter diff comparing to `ipaddr` which will be confusing.

Reply:
```
    Compile5 = {allow, {ipaddrs,[{{127,0,0,1},{127,0,0,1},32},
```
what does this mean ? `"127.0.0.1/32"` ? if it's parsed to a two-element tuple, why should we still need to keep `32` ?

Reply:
I think this is ok. ipaddrs and ipaddr correspond to each other so it is better understood?

Reply:
Yes, the last digit is not required. But in this PR, we should not modify it in order to maintain compatibility.

(This format is derived from the definition in the esockd_cidr module. If you want to change it, you should put it in esockd 6.x)

Reply:
I thought this is also supported in our config interface.
I’m ok as long as we only support string format in config and keep this 3-tuple transparent.

Hope there’s a better way to test without having to inspect this opaque